### PR TITLE
asp: add the PersonnePhysique XML entity

### PR DESCRIPTION
**Pull request by freesteph**
Here's the gameplan: we're going to leverage the Active Model API to encode some of the ASP logic into the app rather than doing some on-the-fly mapping.

This means backporting a bunch of the SERAPIS logic to plain old Ruby objects (albeit extending Active Model) that:

1. declare attributes and possible validations ;
2. are able to be instantiated/mapped from our Active Record models ;
3. declare a `to_xml` method that can tack onto an existing builder object.

This should enable some form of validation chain to make sure we don't send garbage over.

### CODE REVIEW

File: app/services/asp/entities/personne_physique.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        prenom
+        datenaissance
+        codeinseepaysnai
+        codeinseecommune
+      ]
+
+      def self.from_student(student)
+        mapper = ASP::Mappers::StudentMapper.new(student)
+
+        new.tap do |model|
```
Comment (JeSuisUnCaillou) [inline app/services/asp/entities/personne_physique.rb:33]: Je dirais plutôt `instance` que `model` ici

File: app/services/asp/mappers/student_mapper.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      def initialize(student)
+        @student = student
+      end
+
+      MAPPING.each do |name, attr|
+        define_method(name) { student[attr] }
+      end
+
+      def titre
+        GENDER_FOR_SEX[student.biological_sex] || "0" # FIXME: remove
```
Comment (JeSuisUnCaillou) [inline app/services/asp/mappers/student_mapper.rb:36]: "FIXME: remove" ?

**Review by JeSuisUnCaillou — APPROVED** [review]
Juste une remarque sur un naming, et y'a un commentaire qui traîne

File: app/services/asp/entities/personne_physique.rb (hunk 1 above)
Comment (freesteph) [reply app/services/asp/entities/personne_physique.rb]: done

File: app/services/asp/mappers/student_mapper.rb (hunk 2 above)
Comment (freesteph) [reply app/services/asp/mappers/student_mapper.rb]: done

**Pull request merged**